lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -173,6 +173,7 @@ WasmBytecodeGenerator::GenerateFunction()
     m_currentFunc->wasmInfo = m_funcInfo;
     m_nestedIfLevel = 0;
     m_maxArgOutDepth = 0;
+    m_evalStack.Clear();
```

Comment:
This can happen if another function throws an exception and we continue to compile other functions.
I could do the cleanup in the exception case and have an assert here. What do you think ?